ui/src/app/machines/utils.ts

```diff
@@ -18,7 +21,7 @@ export const canOpenActionForm = (
   // select the destination machines first then when the form is open we select
   // the machine to actually perform the clone action.
   if (actionName === NodeActions.CLONE) {
-    return true; // TODO - add basic validation for what can be cloned to.
+    return true;
```

Comment:
It was decided that we wouldn't handle client-side validation for what can and can't be cloned to - the error messages should suffice.